processor/ratelimitprocessor/gubernator.go

```diff
@@ -126,6 +164,12 @@ func (r *gubernatorRateLimiter) RateLimit(ctx context.Context, hits int) error {
 				zap.String("processor_id", r.set.ID.String()),
 				zap.Strings("metadata_keys", r.cfg.MetadataKeys),
 			)
+			r.requestTelemetry(ctx, []attribute.KeyValue{
+				telemetry.WithProcessorID(r.set.ID.String()),
+				telemetry.WithReason(telemetry.StatusOverLimit),
```

Comment:
Isn't overlimit same as too many requests? Isnt throttled already covers that we are already over the limit? I feel we can simplify this and just remove the reason when we are throttled.

Reply:
Yep totally will do. I only added it so all metrics have the same attributes. I don't think it's necessary!